entrypoint.sh

```diff
@@ -38,6 +38,11 @@ if [ "$MAILNAME" ]; then
 	echo "$MAILNAME" > /etc/mailname
 fi
 
+if [ "SMTPPORTOUT" ]; then
```

Comment:
The test failed because of this. I expect it's a typo and it should be:
```suggestion
if [  "${SMTPPORTOUT}"  ]; then
```